src/main/java/com/jagrosh/jmusicbot/commands/dj/ForceRemoveCmd.java

```diff
@@ -0,0 +1,60 @@
+package com.jagrosh.jmusicbot.commands.dj;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.commands.MusicCommand;
+
+
+public class ForceRemoveCmd extends MusicCommand {
+    public ForceRemoveCmd(Bot bot) {
+        super(bot);
+        this.name = "forceremove";
+        this.help = "removes all entries by the mentioned user from the queue";
+        this.arguments = "<@user|id>";
+        this.aliases = new String[]{"forcedelete"};
+        this.beListening = true;
```

Comment:
beListening should be `false`; mods shouldn't have to be actively listening to be able to remove a user's tracks